test/parse.js

```diff
@@ -16,18 +16,22 @@ describe( 'BlockMap.parse()', function() {
   })
 
   context( 'when input is a string', function() {
-    it( 'parses valid data', function() {
-      var json = require( `./data/version-2.0` )
-      var xml = fs.readFileSync( path.join( __dirname, `/data/version-2.0.bmap` ) )
-      assert.deepEqual( json, BlockMap.parse( xml ) )
+    BlockMap.versions.forEach( function( v ) {
+      it( 'parses v' + v, function() {
+        var json = require( `./data/version-${v}` )
+        var xml = fs.readFileSync( path.join( __dirname, `/data/version-${v}.bmap` ) )
```

Comment:
use path.join , remove slashes

Reply:
That's already happening in https://github.com/resin-io-modules/blockmap/pull/34 – will rebase then